Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs"); // Import the File System module
+
+// Read the file content
+const input = fs.readFileSync("input.txt", "utf8"); // 'utf8' ensures the data is read as a string
+
+console.log("Current directory:", process.cwd());
+
+const inputArray = input.split("\n");
+console.log(inputArray);
+
+const numberArray = inputArray.map(Number);
```

Comment:
Can you explain what the expression `inputArray.map(Number)` does? Why do you pass `Number` as the parameter to `inputArray.map()`?

Reply:
When Number is passed as the callback to map(), it is invoked on each element of the array.
For each element x in inputArray, Number(x) is called, converting x to a number, and the result is added to the new array.